pre_test_hook.sh

```diff
@@ -48,13 +48,9 @@ install_python_pkg a10-openstack-lib "$A10_OPENSTACK_LIB_GIT"
 install_python_pkg a10-neutron-lbaas "$A10_NEUTRON_LBAAS_GIT"
 
 set -e
-
 if [ "$testenv" != "apiv1" ]; then
     cat > $DEVSTACK_PATH/local.conf <<EOF
-[[post-config|\$NEUTRON_LBAAS_CONF]]
-
-[service_providers]
-service_provider=LOADBALANCERV2:A10Networks:neutron_lbaas.drivers.a10networks.driver_v2.ThunderDriver:default
+NEUTRON_LBAAS_SERVICE_PROVIDERV2="LOADBALANCERV2:A10Networks:neutron_lbaas.drivers.a10networks.driver_v2.ThunderDriver:default"
```

Comment:
If you really want to get fancy, switch to using the in-repo devstack plugin.  These scripts pre-date that plugin.